doc/plugins.md

```diff
@@ -152,6 +152,8 @@ Have a good idea for a new plugin?  Let’s [chat][gitter] and make it happen!
     — Transform links and images to references and definitions
 *   [`remark-rehype`](https://github.com/remarkjs/remark-rehype)
     — [rehype](https://github.com/rehypejs/rehype) support
+*   [`remark-relative-links`](https://github.com/zslabs/remark-relative-links)
+    - Rewrites absolute URLs to relative ones
```

Comment:
Could you use an em dash here (`—`), instead of the dash? The dash makes the description a list item as well!